zenodo/modules/stats/exporters.py

```diff
@@ -103,7 +103,7 @@ def run(self, start_date=None, end_date=None, update_bookmark=True):
                     'begin_event_timestamp': event_chunk[0].timestamp,
                     'end_event_timestamp': event_chunk[-1].timestamp,
                 }
-                raise PiwikExportRequestError(msg, export_info=info)
+                raise PiwikExportRequestError(info, msg)
```

Comment:
It's better to be explicit and pass the info as a keyword parameter (i.e. `export_info=info`), since in the constructor of the `PiwikExportRequestError` it's "consumed" as such.